pages/DetailsTrip/[city].js

```diff
@@ -0,0 +1,122 @@
+import { useRouter } from "next/router";
+import { v4 as uuidv4 } from "uuid";
+import {
+  StyleButton,
+  StyleList,
+  StyleDiv,
+  StyleCountry,
+  StyleSection,
+  StyleSpan,
+  StyleH4,
+  StyleTotalPrice,
+} from "../../src/components/StyledTripDetails";
+import Header from "@/src/components/Header/Header";
+import { countries } from "@/lib/db";
+
+export default function DetailsOfTrip() {
+  const router = useRouter();
+  const cityName = router.query.city;
+
+  const country = countries.find((country) =>
+    country.cities.find((city) => city.name === cityName)
+  );
+
+  const city = country?.cities.find((city) => city.name === cityName);
+
+  if (!city) {
+    return <h2>City not found</h2>;
+  }
+
+  const totalHotelPrice = city.hotel.map((hotel) => hotel.price);
+  const totalPriceOfHotels = totalHotelPrice.reduce(
+    (hotelPrice, price) => hotelPrice + Number(price),
+    0
+  );
+
+  const totalPlacePrice = city.place.map((place) => place.price);
+  const totalPriceOfPlaces = totalPlacePrice.reduce(
+    (placePrice, price) => placePrice + Number(price),
+    0
+  );
+
+  const totalFoodPrice = city.food.map((food) => food.price);
+  const totalPriceeOfFoods = totalFoodPrice.reduce(
```

Comment:
I think it should be totalPriceOfFoods, seems like there's an additional e. 🤔 But you did stick to the typo from here on out, so be careful. 😂